src/js/components/DescriptionList.js

```diff
@@ -1,6 +1,12 @@
+import PureRenderMixin from 'react-addons-pure-render-mixin';
 import React from 'react';
 
 class DescriptionList extends React.Component {
+  constructor() {
+    super(...arguments);
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate;
```

Comment:
Yep, absolutely - and it does have access to `this.props` and `this.state` when it's called. I'm just trying to figure out why that example uses binding - do you know of any circumstances where it's invoked in the wrong scope?